packages/gestalt/src/SearchField.js

```diff
@@ -78,104 +67,116 @@ export default class SearchField extends React.Component<Props, State> {
+            __style: {
+              pointerEvents: 'none',
+              // Added the following lines for Safari support
+              top: '50%',
+              transform: 'translateY(-50%)',
+            },
+          }}
+          left
+          paddingX={4}
+          position="absolute"
+        >
+          <Icon icon="search" accessibilityLabel="" />
+        </Box>
+      )}
+      <input
+        ref={forwardedRef}
+        aria-label={accessibilityLabel}
+        autoComplete={autoComplete}
+        className={className}
+        id={id}
+        onChange={handleChange}
+        placeholder={placeholder}
+        role="searchbox"
+        type="search"
+        value={value}
+      />
+      {hasValue && (
+        <button
+          className={styles.clear}
+          onClick={handleClear}
+          tabIndex={-1}
+          type="button"
+        >
           <Box
-            dangerouslySetInlineStyle={{
-              __style: {
-                pointerEvents: 'none',
-                // Added the following lines for Safari support
-                top: '50%',
-                transform: 'translateY(-50%)',
-              },
-            }}
-            left
-            paddingX={4}
-            position="absolute"
+            alignItems="center"
+            color={focused ? 'darkGray' : 'transparent'}
+            display="flex"
+            height={clearButtonSize}
+            justifyContent="center"
+            rounding="circle"
+            width={clearButtonSize}
           >
-            <Icon icon="search" accessibilityLabel="" />
+            <Icon
+              accessibilityLabel=""
+              color={focused ? 'white' : 'darkGray'}
+              icon="cancel"
+              size={clearIconSize}
+            />
           </Box>
-        )}
-        <input
-          aria-label={accessibilityLabel}
-          autoComplete={autoComplete}
-          className={className}
-          id={id}
-          onChange={this.handleChange}
-          placeholder={placeholder}
-          role="searchbox"
-          type="search"
-          value={value}
-        />
-        {hasValue && (
-          <button
-            className={styles.clear}
-            onClick={this.handleClear}
-            tabIndex={-1}
-            type="button"
-          >
-            <Box
-              alignItems="center"
-              color={focused ? 'darkGray' : 'transparent'}
-              display="flex"
-              height={clearButtonSize}
-              justifyContent="center"
-              rounding="circle"
-              width={clearButtonSize}
-            >
-              <Icon
-                accessibilityLabel=""
-                color={focused ? 'white' : 'darkGray'}
-                icon="cancel"
-                size={clearIconSize}
-              />
-            </Box>
-          </button>
-        )}
-      </Box>
-    );
-  }
+        </button>
+      )}
+    </Box>
+  );
+};
+
+SearchField.displayName = SearchField;
```

Comment:
@simkessy are you confident this works, shouldn't this be a string instead? `SearchField.displayName = 'SearchField'`